src/dense/stiff_addsteps.jl

```diff
@@ -350,7 +350,7 @@ function _ode_addsteps!(k, t, uprev, u, dt, f, p,
         always_calc_begin = false, allow_calc_end = true,
         force_calc_end = false)
     if length(k) < 2 || always_calc_begin
-        @unpack du, du1, du2, tmp, k1, k2, k3, k4, k5, k6, dT, J, W, uf, tf, linsolve_tmp, jac_config, fsalfirst = cache
+        @unpack du, du1, du2, tmp, k1, k2, k3, k4, k5, dT, J, W, uf, tf, linsolve_tmp, jac_config, fsalfirst = cache
```

Comment:
Maybe we should add a test for this part?

Reply:
k6 is not present in the cache of Rodas23W, Rodas3P, this was an error.